test/mysql_handler/start

```diff
@@ -10,7 +10,11 @@ done
 
 ./stop
 
+# ./ssl is REQUIRED during the build phase to make test ssl cert artifacts available to the docker context
```

Comment:
Let's add a note about how the certs are stored in util/ssl beause the same ones are shared, and we copy them here only while the test is running.